### PR TITLE
[IMP] event: Improve confirmation and reminder event mail templates

### PR DESCRIPTION
**Pull request by jat-odoo**
Description of the issue/feature this PR addresses:
- Task: https://www.odoo.com/web#id=26672&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
- Pad: https://pad.odoo.com/p/r.a2f7fda2574122099847dc66a52cb3dd

Current behavior before PR:

Desired behavior after PR is merged:
## 

I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/event/data/email_template_data.xml [hunk 1]
```diff
@@ -29,11 +29,112 @@
...
+                                <a href="/event" style="text-decoration:none;color:#717188;">More Events</a>
+                            </td>
+                        </tr>
+                    </tbody>
+                </table>
+                <table style="width:600px;margin:0px auto;background:white;border:1px solid #e1e1e1;">
+                    <tbody>
+                        <tr>
+                            <td style="padding:15px 20px 0px 20px;">
+                                <h2 style="font-weight:300;">Hi ${object.name}, this is your order confirmation for <a href="/event/${object.event_id.id}" style="color:#a24689;text-decoration:none;">${object.event_id.name}</a></h2>
```
Comment (bst-odoo) [inline addons/event/data/email_template_data.xml]: Instead of "this is", "here is" is better.

File: addons/event/data/email_template_data.xml [hunk 2]
```diff
@@ -43,17 +144,123 @@
...
+                                            <div style="font-size:12px;color:#9e9e9e"><i><strong>TZ</strong> ${object.event_id.date_tz}</i></div>
+                                        </td>
+                                    </tr>
+                                </table>
+                                % if object.event_id.organizer_id.email:
+                                    <table style="width:100%;border-top:1px solid #e1e1e1;">
+                                        <tr>
+                                            <td style="padding:10px 0px 25px 0px;">
+                                                <h2 style="font-weight:300;margin:10px 0px">Questions about this event?</h2>
+                                                <div>Contact organizer at <a href="mailto:${object.event_id.organizer_id.email}" style="text-decoration:none;color:#a24689;">${object.event_id.organizer_id.email}</a></div>
```
Comment (bst-odoo) [inline addons/event/data/email_template_data.xml]: Please contact the organizer at ...

File: addons/event/data/email_template_data.xml [hunk 3]
```diff
@@ -43,17 +144,123 @@
...
+                    </tbody>
+                </table>
+                <table style="width:600px;margin:0px auto;background:white;border:1px solid #e1e1e1;">
+                    <tbody>
+                        <tr>
+                            <td style="padding:15px 20px 0px 20px;">
+                                <h2 style="font-weight:300;">
+                                    Hi ${object.name}, 
+                                    % if object.reminder_text:
+                                        You have
```
Comment (bst-odoo) [inline addons/event/data/email_template_data.xml]: Please keep in mind that the event *event name* takes place on *this monday*

File: addons/event/data/email_template_data.xml [hunk 4]
```diff
@@ -29,11 +29,112 @@
...
+                                                <h2 style="font-weight:300;margin:10px 0px">Questions about this event?</h2>
+                                                <div>Contact organizer at <a href="mailto:${object.event_id.organizer_id.email}" style="text-decoration:none;color:#a24689;">${object.event_id.organizer_id.email}</a></div>
+                                            </td>
+                                        </tr>
+                                    </table>
+                                % endif
+                                <table style="width:100%;border-top:1px solid #e1e1e1;">
+                                    <tr>
+                                        <td style="padding:25px 0px;">
+                                            <strong> Add this event to calender</strong>
```
Comment (bst-odoo) [inline addons/event/data/email_template_data.xml]: your calendar

File: addons/event/data/email_template_data.xml [hunk 5]
```diff
@@ -43,17 +144,123 @@
...
```
Comment (bst-odoo) [inline addons/event/data/email_template_data.xml]: your calendar

**Comment by bst-odoo** [comment]
Watch out for the typos : calendar is written with a, not calender. Thx.

File: addons/event/models/event.py [hunk 6]
```diff
@@ -261,6 +278,14 @@ def mail_attendees(self, template_id, force_send=False, filter_func=lambda self:
         for attendee in self.registration_ids.filtered(filter_func):
             self.env['mail.template'].browse(template_id).send_mail(attendee.id, force_send=force_send)
 
+    # In event mail template need to check is website_event is installed or not?
+    # If only event is installed than display name of event in mail template,
+    # If website_event is installed than display link to event in mail template
+    url = fields.Char(compute='_compute_url')
```
Comment (tde-banana-odoo) [inline addons/event/models/event.py]: Nope.

File: addons/event/models/event.py [hunk 7]
```diff
@@ -408,3 +434,28 @@ def action_send_badge_email(self):
...
+            today = fields.datetime.now().replace(hour=0, minute=0, second=0)
+            event_date = fields.Datetime.from_string(attendee.event_begin_date)
+            next_month = today + relativedelta(months=1)
+            month_range = calendar.monthrange(next_month.year, next_month.month)
+            diff = event_date - today
+            if diff.days < 0:
+                return
+            if diff.days < 7:
+                if diff.days == 0:
+                    attendee.reminder_text = 'Today'
```
Comment (tivisse) [inline addons/event/models/event.py]: Translation in this ?

File: addons/event/models/event.py (hunk 6 above)
Comment (jat-odoo) [inline addons/event/models/event.py]: @tde-banana-odoo ,
Any better way to do this ?

**Pull request merged**